src/ts/contentScripts/dispatcher/index.ts

```diff
@@ -20,6 +20,20 @@ const dispatchMap = new Map([
 
 browser.runtime.onMessage.addListener((command) => dispatchMap.get(command)?.());
 
+document.addEventListener('keydown', ev => {
+    const enter = 'Enter';
+    const isExitingTitle = (ev: KeyboardEvent) => {
+        return ev.target.parentElement instanceof HTMLHeadingElement;
```

Comment:
given that you define this within the context of the handler, the event is already in the closure, so you don't have to pass it to the function.   
in fact you can probably just use variable instead of function..